.github/workflows/build_and_test.yml

```diff
@@ -5,11 +5,46 @@ on:
     branches:
     - '**'
     - '!branch-*.*'
+  schedule:
+    # master
+    - cron: '0 0 * * *'
```

Comment:
Could you shift this more? `0 0 * * *` is used at `Publish Snapshot` job already.